### PR TITLE
Added preliminary testing for behave and bug fixes

### PR DESCRIPTION
**Pull request by sean-lewis**

### CODE REVIEW

File: core/features/environment.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+def before_all(context):
+
+    setup_debug_on_error(context.config.userdata)
+
+    desired_capabilities = webdriver.DesiredCapabilities.FIREFOX
+    desired_capabilities['version'] = '12'
+    desired_capabilities['platform'] = 'WINDOWS'
+    desired_capabilities['name'] = 'Testing Col-League App'
+    desired_capabilities['client_key'] = '44fa53ac66807797ca8ec3f1e9f9a9d9'
+    desired_capabilities['client_secret'] = '5cc0cce8a93b22b347ffd72204d007bb'
```
Comment (kave) [inline core/features/environment.py:20]: whats this?

File: core/features/steps/steps.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@when('we visit col-league')
+def step(context):
+    context.browser.get('http://cole-league.herokuapp.com')
```
Comment (kave) [inline core/features/steps/steps.py]: typo: col-league

File: core/features/environment.py (hunk 1 above)
Comment (sean-lewis) [inline core/features/environment.py:20]: It is the beginning creation of behave. It is client keys to the behave testing site I setup. It doesnt do anything yet but I wanted to get some stuff down

File: core/features/steps/steps.py (hunk 2 above)
Comment (sean-lewis) [inline core/features/steps/steps.py]: Thank you!

File: core/features/environment.py (hunk 1 above)
Comment (kave) [inline core/features/environment.py:20]: @madridsoccer5 aaah i see, so its a behave "server" running locally?

File: README.md [hunk 3]
```diff
@@ -16,10 +16,18 @@ Setup
...
 * Launch the app - `./manage.py runserver 0.0.0.0:8000`
 * Access the app in your browser - `http://localhost:8111`
+
+
+Running Behave
+--------------
+*To run selenium with behave:
+* 1) Setup a free account on [testingbot](http://testingbot.com/)
+* 2) Add client secret and key in face_it/core/environment.py
+* 3) Go to the core folder on the command line and run "behave"
```
Comment (kave) [inline README.md:33]: nice

**Pull request merged**